Dockerfile

```diff
@@ -1,5 +1,6 @@
 # use the ui container to pull in webpack artifacts
-FROM containers.renci.org/helxplatform/helx-ui:v3.0.0 as builder
+# FROM containers.renci.org/helxplatform/helx-ui:v3.0.0 as builder
+FROM containers.renci.org/helxplatform/helx-ui:external-app-registry as builder
```

Comment:
why does this depend on a specific tag instead of the release tag? IMO we shouldn't be changing the base helx-ui image to something other than a release tag in develop.

Reply:
The changes in the 'external-app-registry' branch of helx-ui are needed for the changes in the appstore branch to work.